source/backend/dal/Repositories/CompensationRequisitionRepository.cs

```diff
@@ -182,5 +182,13 @@ public List<PimsPropertyLease> GetLeaseCompReqPropertiesById(long compensationRe
                 .Select(x => x.PropertyLease)
                 .ToList();
         }
+
+        public IEnumerable<PimsCompReqFinancial> GetCompensationRequisitionFinancials(long id)
+        {
+            return Context.PimsCompReqFinancials
+                .Include(y => y.FinancialActivityCode)
+                .Where(x => x.CompensationRequisitionId == id)
+                .ToList();
+        }
```

Comment:
I think this method is only ever used to "read" data from the DB so it would make sense to add **.AsNoTracking()** - thoughts?

Reply:
agree